src/vunnel/providers/mariner/model/generated.py

```diff
@@ -270,13 +302,15 @@ class Meta:
         default=None,
         metadata={
             "type": "Attribute",
-        }
+            "required": True,
+        },
     )
-    criterion: Optional[Criterion] = field(
-        default=None,
+    criterion: List[Criterion] = field(
```

Comment:
This is the important change to the generated model. It might be worth backing out all. the `required: True`, since it's noisy and might lead to a failure to parse the whole OVAL file if there's one malformed record. (The intended behavior of Vunnel is to skip malformed records but carry on and still generate some data.)

My understanding is that right now there will always be exactly 2 items on the list, at least right now.

Reply:
This Vunnel parser is written to ignore the failure to parse individual definitions anyway: https://github.com/anchore/vunnel/blob/main/src/vunnel/providers/mariner/parser.py#L45

Based on some experimentation, the new `required: True` annotations are showing up because of bumping the `xsdata` package to version 23.5.

Reply:
Based on offline discussion, we will pin `xsdata` package back to the version that generated these models to avoid introducing unnecessary drift in the models. (Putting `required: True` everywhere has no benefit on a field of type `Optional`, since given a populated model we have to check for `None`, and failing validation for this reason would be unfortunate, since the code defends against these `None`s anyway.)